templates/tools/dockerfile/python_pip_builds.include

```diff
@@ -0,0 +1,11 @@
+#===================================
+# Install Python build requirements
+# TODO: Update cython bounds as we are using cython 3.0.0 now
+RUN /opt/python/cp36-cp36m/bin/pip install --upgrade 'cython<4.0.0rc1'
+RUN /opt/python/cp37-cp37m/bin/pip install --upgrade 'cython<4.0.0rc1'
```

Comment:
Do we still need 3.6 and 3.7?

Reply:
we are not building 3.6 and 3.7 Python artifacts, so I believe it can safely be removed.